src/Chapter2.hs

```diff
@@ -336,7 +348,13 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList x y zs
+  | x < 0 = []
+  | y < 0 = []
+  | null zs = []
+  | x > y = []
+  | x == 0    = head zs : subList x (y-1) (tail zs)
+  | otherwise = subList (x-1) (y-1) (tail zs)
```

Comment:
Using `head` and `tail` is not efficient. You can take a look at `take` and `drop` functions that could help you here 🙂